src/faebryk/library/can_query_jlcpcb_db.py

```diff
@@ -0,0 +1,12 @@
+# This file is part of the faebryk project
```

Comment:
```python
class has_multi_picker(has_picker.impl()):
    def pick(self):
        for prio, picker in self.pickers:
            # TODO handle failure / success
            picker.pick()

    class Picker:
        @abstractmethod
        def pick(self): ...

    def __init__(self) -> None:
        super().__init__()

        self.pickers: list[tuple[int, has_multi_picker.Picker]] = []

    def add_picker(self, prio: int, picker: Picker):
        self.pickers.append((prio, picker))
        self.pickers = sorted(self.pickers, key=lambda x: x[0])

    @classmethod
    def add_to_module(cls, module: Module, prio: int, picker: Picker):
        if not module.has_trait(has_picker):
            module.add_trait(cls())

        t = module.get_trait(has_picker)
        assert isinstance(t, has_multi_picker)
        t.add_picker(prio, picker)

```

Reply:
I'm not really sure about how I implemented the use of `has_multi_picker.Picker` in `src/faebryk/libs/picker/jlcpcb/pickers.py`, you should take a look